Assets/Scripts/SS3D/Systems/Inventory/Items/Item.cs

```diff
@@ -100,10 +99,16 @@ public void Init(string name, float weight, Vector2Int size,  List<Trait> traits
         /// <summary>
         /// The sprite that is shown in the container slot
         /// </summary>
-
         public Sprite Sprite
         {
-            get => InventorySprite();
+            get
```

Comment:
I think john rather have a method to call in the getter instead of the long getter, put that back in its initial state.